SPECS-EXTENDED/dovecot/dovecot.spec

```
@@ -164,8 +163,8 @@ autoreconf -I . -fiv #required for aarch64 support
     --with-ssl=openssl           \
     --with-ssldir=%{ssldir}      \
     --with-solr                  \
-    --with-systemdsystemunitdir=%{_unitdir}  \
-    --with-docs
+    --with-docs                  \
+    systemdsystemunitdir=%{_unitdir}
```

Are we missing a `--`:
```suggestion
    --with-systemdsystemunitdir=%{_unitdir}
```

---

No. It fails to compile with that a tag like structure now.